### PR TITLE
docs: Update NFT use case for latest subnets version

### PR DESCRIPTION
**Pull request by jbencin**
### Description

Update `getting-started.md` to be correct for recent changes

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -216,31 +216,37 @@ Open the new file and add the following content:
...
 
 The settings for the devnet are found in _./settings/Devnet.toml_. In order to launch a subnet in the devnet, we need to tell Clarinet to enable a subnet node and a corresponding API node.
 
-Uncomment or add the following line under `[devnet]`:
+#### Required Changes
+
+Uncomment or add the following lines under `[devnet]`:
 
 ```toml
+# By default, subnets are off
```
Comment (obycode) [inline docs/getting-started.md:228]:
```suggestion
# By default, the subnet is not enabled
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -216,31 +216,37 @@ Open the new file and add the following content:
...
 enable_subnet_node = true
+# Currently required to use subnets with API
```
Comment (obycode) [inline docs/getting-started.md:230]:
```suggestion
# Currently required to use an API node with the subnet
```

File: docs/getting-started.md [hunk 3]
```diff
@@ -216,31 +216,37 @@ Open the new file and add the following content:
...
+stacks_api_env_vars = ["STACKS_CHAIN_ID=2147483648", "CUSTOM_CHAIN_IDS=testnet=0x55005500"]
```
Comment (obycode) [inline docs/getting-started.md:231]: Is the `STACKS_CHAIN_ID=2147483648` definitely required? I thought that was the default, and I thought that we needed to set `STACKS_CHAIN_ID=0x55005500` for the subnet API node?

Comment (jbencin) [reply docs/getting-started.md]: I'm just going to specify a newer, unreleased version of Clarinet to avoid having to explain this

**Review by obycode — APPROVED** [review]
LGTM

**Pull request merged**